sunpy/map/tests/test_mapbase.py

```diff
@@ -75,23 +92,30 @@ def test_fits_data_comparison(aia171_test_map):
 
 def test_get_item(generic_map):
     with pytest.raises(NotImplementedError):
-        generic_map[10,10]
+        generic_map[10, 10]
 
 
 def test_wcs(aia171_test_map):
     wcs = aia171_test_map.wcs
     assert isinstance(wcs, astropy.wcs.WCS)
 
-    assert all(wcs.wcs.crpix == [aia171_test_map.reference_pixel.x.value,
-                                 aia171_test_map.reference_pixel.y.value])
-    assert all(wcs.wcs.cdelt == [aia171_test_map.scale.x.value,
-                                 aia171_test_map.scale.y.value])
-    assert all(wcs.wcs.crval == [aia171_test_map.reference_coordinate.x.value,
-                                 aia171_test_map.reference_coordinate.y.value])
-    assert set(wcs.wcs.ctype) == set([aia171_test_map.coordinate_system.x,
-                             aia171_test_map.coordinate_system.y])
+    assert all(wcs.wcs.crpix == [
+        aia171_test_map.reference_pixel.x.value,
+        aia171_test_map.reference_pixel.y.value
+    ])
```

Comment:
I prefer the indentation that we had previously, this is not easier to read.
If you are going to close a list in the following line, then I would prefer the last element of the list to be followed by a comma, so if a new item is added git won't see two modifications, but only one.
```python
a = [ 
   1,
   2,
  ]
```